DEVELOPMENT.md

```diff
@@ -0,0 +1,92 @@
+# Development
+This file highlight the installation process of getting started with Ersilia, Ersilia Artifacts and Creating a PR.
+
+## Getting Started
+Ersilia can be run either on local computer or with the use of GitHub codespace. On your local computer, ersilia is designed to run specific environment
+namely: **Linux and Linux-like environment** (MacOS, WSL).
+
+- ### Installation of Ersilia on local computer
+To get started with Ersilia installation, there are few dependencies that needs to be installed before proceeding to installation of Ersilia. Please check the package requirement
+in the [Installation Guide](https://ersilia.gitbook.io/ersilia-book/ersilia-model-hub/installation)
+1. Once the required dependencies has been met, open a terminal and set up a Conda environment.
+
+```
+# create a conda environment
+conda create -n ersilia python=3.10
+# activate the environment
+conda activate ersilia
+```
+2. Clone the repository and run to install the Ersilia Python package.
+```
+# clone from github
+git clone https://github.com/ersilia-os/ersilia.git
+cd ersilia
+# install with pip (use -e for developer mode)
+pip install -e .
```

Comment:
We should instead ask developers to install it as `pip install -e .[test]` instead.